### DISCUSSION TITLE
Add falling leaves animation when🍃emoji is sent

### DISCUSSION THREAD

**Discussion by ghost** [element-hq/element-meta#549] [Ideas]
#### What would you like to do?

Do an animation of falling leaves when🍃is sent or add a "/leaves" command.

#### Why would you like to do it?

It would enhance the experience

#### How would you like to achieve it?

Do it in a similar way as 🌧️ or 💝 but it would be leaves.

**Comment by daniellekirkwood** [comment]
Thanks for the suggestion! 

We're currently not looking to add any more chat effects to our product as we have 6 already, any more runs the risk of being overpowering. [See more info here](https://github.com/vector-im/element-meta/blob/develop/docs/chat_effects.md)

Happy for this to be left as an open discussion as lots of up-votes or comments would certainly ensure we revisit this decision.
